exercises/practice/after-party/.meta/config.json

```
@@ -0,0 +1,20 @@
+{
+    "authors": [
+      "jkwon100"
```

```suggestion
        "iHiD",
        "jkwon100"
```

Jeremy/iHiD authored this exercise :)